model_training/README.md

```diff
@@ -106,20 +136,21 @@ Run
 
 1. Fork `kepler-model-db`.
 
-1. Validate and make a copy by export command. Need to define `machine id`, `local path to forked kepler-model-db/models`, and `author github account`. 
+1. Validate and make a copy by export command. Need to define `machine id`, `local path to forked kepler-model-db/models`, `author github account` and `benchmark type`.
 
     Run
     ```
-    ./script.sh export <machine id> <path to kepler-model-db/models> <author github account>
+    ./script.sh export <machine id> <path to kepler-model-db/models> <author github account> <benchmark type>
```

Comment:
I would like to avoid fixing the benchmark type here since we also expect the mix type of benchmarks (both standard and custom).